src/modules/tools/temperaturecontrol/TemperatureControl.h

```diff
@@ -83,14 +81,17 @@ class TemperatureControl : public Module {
         float PIDdt;
 
         // Temperature runaway values
+        enum RUNAWAY_TYPE {NOT_HEATING, HEATING_UP, COOLING_DOWN, MAINTAINING_TEMPERATURE};
+
         RUNAWAY_TYPE runaway_state;      
+        uint8_t runaway_heating_timer;
+        float runaway_last_temperature;
```

Comment:
do we really need to add another 4 bytes per instance? increasing memory like this is really bad when it is not necessary.